packages/make-dedicated-lockfile/test/fixtures/fixture/packages/is-negative/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "dependencies": {
     "ramda": "0.26.0",
+    "request": "^2.0.0",
```

Comment:
why was this added?

Reply:
I wanted to add a package which had a specifier instead of a set version to test whether the version would be re-resolved when the dedicated lockfile was created